tests/unit/python/foglamp/services/core/test_connect.py

```diff
@@ -0,0 +1,46 @@
+# -*- coding: utf-8 -*-
+
+# FOGLAMP_BEGIN
+# See: http://foglamp.readthedocs.io/
+# FOGLAMP_END
+
+from unittest.mock import MagicMock, patch
+import pytest
+
+from foglamp.services.core.service_registry.service_registry import ServiceRegistry as Service
+from foglamp.services.core.service_registry.exceptions import DoesNotExist
+from foglamp.services.core import connect
+from foglamp.common.storage_client.storage_client import StorageClient
+
+__author__ = "Ashish Jabble"
+__copyright__ = "Copyright (c) 2017 OSIsoft, LLC"
+__license__ = "Apache 2.0"
+__version__ = "${VERSION}"
+
+
+@pytest.allure.feature("unit")
+@pytest.allure.story("services", "core")
+class TestConnect:
+    """ Storage connection"""
+    
+    def setup_method(self):
+        Service._registry = []
+
+    def teardown_method(self):
+        Service._registry = []
+
```

Comment:
Not sure about the use of setup and teardown. Can we get rid of classic xunit-style setup?

Reply:
any specific reason? we should start using them at each level as needed to keep state clean